src/cursor/cur_backup.c

```diff
@@ -205,20 +204,16 @@ __curbackup_close(WT_CURSOR *cursor)
         __wt_verbose(
           session, WT_VERB_BACKUP, "%s", "Releasing resources from forced stop incremental");
         __wt_backup_destroy(session);
-        /*
-         * We need to force a checkpoint to the metadata to make the force stop durable. Without it,
-         * the backup information could reappear if we crash and restart.
-         */
-        cfg[0] = WT_CONFIG_BASE(session, WT_SESSION_checkpoint);
-        cfg[1] = "force=true";
-        /*
-         * Metadata checkpoints rely on read-committed isolation. Use that here no matter what
-         * isolation the caller's session sets for isolation.
-         */
-        WT_WITH_DHANDLE(session, WT_SESSION_META_DHANDLE(session),
-          WT_WITH_METADATA_LOCK(session,
-            WT_WITH_TXN_ISOLATION(
-              session, WT_ISO_READ_COMMITTED, ret = __wt_checkpoint(session, cfg))));
+    }
+
+    /*
+     * For either a force stop or a full backup starting an incremental force a checkpoint so that
+     * the new information is visible in the metadata and old backup information does not reappear
+     * if we crash and restart.
+     */
+    if (F_ISSET(cb, WT_CURBACKUP_FORCE_STOP) ||
+      (F_ISSET(cb, WT_CURBACKUP_INCR) && cb->incr_src == NULL)) {
+        WT_TRET(session->iface.checkpoint(&session->iface, "force=1"));
```

Comment:
Can you please add a comment explaining why this needs to be a forced checkpoint (as opposed to a normal checkpoint)?